### PR TITLE
CHK-1363-auth-request

### PR DESCRIPTION
**Pull request by Fernando-Granato**
### List of changes
Changed http response code from 400 to 504 into auth-request base policy.

### Motivation and context
Changed the policy for pspId check. In case the pspId was not registered on the XPAY or VPOS lists it will return a 504

### Type of changes

- [ ] Add new resources
- [x] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### CODE REVIEW

File: src/domains/ecommerce-app/api/ecommerce-checkout/v1/_auth_request.xml.tpl [hunk 1]
```diff
@@ -45,14 +45,14 @@
             </when>
             <otherwise>
                 <return-response>
-                    <set-status code="400"/>
+                    <set-status code="504"/>
```
Comment (infantesimone) [inline src/domains/ecommerce-app/api/ecommerce-checkout/v1/_auth_request.xml.tpl:48]: why is it a timeout in this case?

**Review by infantesimone — CHANGES_REQUESTED** [review]

File: src/domains/ecommerce-app/api/ecommerce-checkout/v1/_auth_request.xml.tpl (hunk 1 above)
Comment (infantesimone) [reply src/domains/ecommerce-app/api/ecommerce-checkout/v1/_auth_request.xml.tpl:48]: I think the 4XX is correct as the input is not valid

Comment (Fernando-Granato) [reply src/domains/ecommerce-app/api/ecommerce-checkout/v1/_auth_request.xml.tpl:48]: OK. So the task can be closed ?

Comment (Fernando-Granato) [reply src/domains/ecommerce-app/api/ecommerce-checkout/v1/_auth_request.xml.tpl:48]: Or we can discuss it at a refinement

Comment (pietro-tota) [reply src/domains/ecommerce-app/api/ecommerce-checkout/v1/_auth_request.xml.tpl:48]: Maybe a 422 is more suitable for this specific case

Comment (Fernando-Granato) [reply src/domains/ecommerce-app/api/ecommerce-checkout/v1/_auth_request.xml.tpl:48]: can be a good idea! I report your idea on JIRA 🙂

**Pull request closed without merging**